components/Post.tsx

```diff
@@ -0,0 +1,43 @@
+import Button from './Button'
+import NextImage from './NextImage'
+
+// types
+import { PostProps } from '../pages/event'
+
+interface PostCardProps {
+  post: PostProps
+  picture: StaticImageData
+}
+
+const Post = ({ post, picture }: PostCardProps): JSX.Element => {
+  const { title, body } = post
+  return (
+    <div className="flex flex-col max-w-xs mx-auto bg-white shadow-lg rounded-lg h-full">
+      <NextImage
+        classnames="max-w-xs overflow-hidden rounded-lg mb-5"
+        imgStyles="rounded-lg transform hover:scale-105 transition easa-in"
+        src={picture}
+        alt="dummy pic"
+        width={300}
+        height={200}
+        layout="responsive"
+        placeholder="blur"
+      />
```

Comment:
@sithumonline no tail semicolon at the end of each line why is that?

Reply:
CI passing ne...? So no need `;` 🤔

Reply:
> CI passing ne...? So no need `;` 🤔

this is not about CI, as a best practices we should use semicolon, this is not Go

Reply:
JS doesn't always know how to intelligently insert semicolons for you.